### PR TITLE
add pivot example for starlark processor

### PR DESCRIPTION
**Pull request by samhld**
### Required for all PRs:

- [ ] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [ ] Has appropriate unit tests.

### CODE REVIEW

File: plugins/processors/starlark/testdata/pivot.star [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'''
+Pivots a key's value to be the key for another key.
+In this example it pivots the value of key `sensor`
+to be the key of the value in key `value`
+
+Input: measurement,tag_set sensor=001A0,value=111.48
+Output: measurement,tag_set 001A0=111.48
```
Comment (ssoroka) [inline plugins/processors/starlark/testdata/pivot.star:7]: love the input output section. We should make this standard for all examples.

File: plugins/processors/starlark/testdata/pivot.star [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+'''
+ 
+ def apply(metric):
+#   metric.fields[str(metric.fields['sensor'])] = metric.fields['value']
+#   metric.fields.pop('value',None)
+#   metric.fields.pop('sensor',None)
+#   return metric
```
Comment (ssoroka) [inline plugins/processors/starlark/testdata/pivot.star:14]: no need to have it commented out.

Comment (samhld) [reply plugins/processors/starlark/testdata/pivot.star]: Got rid of 'em :)

File: plugins/processors/starlark/testdata/pivot.star (hunk 1 above)
Comment (samhld) [reply plugins/processors/starlark/testdata/pivot.star]: Got rid of the tags altogether as they don't play a role here

File: plugins/processors/starlark/testdata/pivot.star [hunk 3]
```diff
@@ -0,0 +1,14 @@
+'''
+Pivots a key's value to be the key for another key.
+In this example it pivots the value of key `sensor`
+to be the key of the value in key `value`
+
+Input: temperature sensor=001A0,value=111.48
+Output: temperature 001A0=111.48
```
Comment (ssoroka) [inline plugins/processors/starlark/testdata/pivot.star:7]:
```suggestion
Input: 
temperature sensor=001A0,value=111.48

Output: 
temperature 001A0=111.48
```

**Pull request merged**